api/book/models.py

```diff
@@ -9,4 +9,5 @@ class Book(SQLModel, table=True):
     name: str = Field()
     author: str = Field()
     type: str = Field()
-    price: float = Field()
\ No newline at end of file
+    price: float = Field()
+    quantity: int = Field()
```

Comment:
I think you can add restriction, like quiantity can't be less than zero. Would be great